app/src/main/java/com/owncloud/android/ui/activity/DrawerActivity.java

```
@@ -1229,60 +1229,60 @@ public void onAccountRemovedEvent(AccountRemovedEvent event) {
      * Retrieves external links via api from 'external' app
      */
     public void fetchExternalLinks(final boolean force) {
-        if (getBaseContext().getResources().getBoolean(R.bool.show_external_links)) {
-            Thread t = new Thread(() -> {
-                // fetch capabilities as early as possible
-                if ((getCapabilities() == null || getCapabilities().getAccountName().isEmpty())
-                    && getStorageManager() != null) {
+        if (!getBaseContext().getResources().getBoolean(R.bool.show_external_links)) {
+            return;
+        }
+
+        Thread t = new Thread(() -> {
+            User user = accountManager.getUser();
+            if (user.isAnonymous()) {
+                accountManager.startAccountCreation(this);
```

as discussed, please keep change as little as possible and only fail silently.
-> this method opens up AuthenticatorActivity.
As we do not know why this is happening, it might be that then the user ends up in an endless loop
- open app
- fetch external link
- anonym -> AuthActivity
- enter server url
- fetch link
- -> step 1

So please remove all, but simply fail if no account can be used to fetch credentials or external links